Investigate failing unit test that bypassed CircleCI

The output from a run of our unit tests includes a lot of spam. Mixed in with that spam, it looks like there are some cases where the test legitimately failed (a chai assertion didn't pass), but the error went to stderr without making the test officially fail. For example:

```
W20181031-00:44:22.359(0)? (STDERR) { AssertionError: expected 0.498037 to equal 0
W20181031-00:44:22.359(0)? (STDERR)     at Assertion.assertEqual (packages/practicalmeteor_chai.js:1420:12)
W20181031-00:44:22.360(0)? (STDERR)     at Assertion.ctx.(anonymous function) (packages/practicalmeteor_chai.js:3662:25)
W20181031-00:44:22.360(0)? (STDERR)     at doAsserterAsyncAndAddThen (/home/circleci/app/lesswrong/node_modules/chai-as-promised/lib/chai-as-promised.js:289:22)
W20181031-00:44:22.360(0)? (STDERR)     at Assertion.<anonymous> (/home/circleci/app/lesswrong/node_modules/chai-as-promised/lib/chai-as-promised.js:255:20)
W20181031-00:44:22.360(0)? (STDERR)     at Assertion.ctx.(anonymous function) [as equal] (packages/practicalmeteor_chai.js:4564:33)
W20181031-00:44:22.360(0)? (STDERR)     at Promise.asyncApply (packages/local-test:lesswrong/testing/voting.tests.js:32:37)
W20181031-00:44:22.360(0)? (STDERR)     at /home/circleci/.meteor/packages/promise/.0.11.1.lt6vo5.rl7ft++os+web.browser+web.browser.legacy+web.cordova/npm/node_modules/meteor-promise/fiber_pool.js:43:40
W20181031-00:44:22.360(0)? (STDERR)   message: 'expected 0.498037 to equal 0',
W20181031-00:44:22.360(0)? (STDERR)   showDiff: true,
W20181031-00:44:22.360(0)? (STDERR)   actual: 0.498037,
W20181031-00:44:22.361(0)? (STDERR)   expected: 0 }
```

It looks like that particular unit test "passes" because the assertions are wrapped in a `forEach` which handles the `async`-ness of its contents incorrectly; but actually, it's been failing for a long time.

--- a/packages/lesswrong/testing/voting.tests.js
+++ b/packages/lesswrong/testing/voting.tests.js
@@ -16,23 +16,6 @@ chai.use(chaiAsPromised);
 describe('Voting', async function() {
   describe('batchUpdating', async function() {
     this.timeout(20000)
-    it('updates if post in the future', async function(){
-      const user = await createDummyUser();
-      const currentTime = new Date();
-      const tomorrow = currentTime.getTime()+(1*24*60*60*1000)
-      const in_a_month = currentTime.getTime()+(30*24*60*60*1000)
-      const in_half_an_hour = currentTime.getTime()+(30*60*1000)
-      const in_ten_minutes = currentTime.getTime()+(10*60*1000)
-      const dates = [tomorrow, in_ten_minutes, in_half_an_hour, tomorrow, in_a_month];
-      dates.forEach(async date => {
-        const post = await createDummyPost(user, {postedAt: date});
-        await batchUpdateScore({collection: Posts});
-        const updatedPost = await Posts.find({_id: post._id}).fetch();
-
-        updatedPost[0].score.should.equal(0);
-        updatedPost[0].postedAt.should.be.closeTo(date, 1000);
-      })
-    });
     it('does not update if post is inactive', async () => {
       const user = await createDummyUser();
       const yesterday = new Date().getTime()-(1*24*60*60*1000)

